recipes/nibabel/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: nibabel
+  version: "0.11"
+
+source:
+    git_url: https://github.com/nipy/nibabel.git
+    git_tag: 2.0.2
+
+requirements:
+  build:
+    - python
+    - numpy
+  run:
+    - python
+    - numpy
+    - scipy
+test:
+  requires:
+    - nose
+  imports:
+    - nibabel
+about:
+  home: https://nipy.org/nibabel
+  license: MIT
+  license_file: COPYING
+  summary: Python package to access a cacophony of neuro-imaging file formats
+extra:
```

Comment:
Could we please space out these sections? One newline before `extra` should be fine.